lua/entities/sent_tardis/init.lua

```diff
@@ -1625,6 +1625,418 @@ function ENT:TogglePower()
 	return false
 end
 
+local function CheckPP(ply, ent) // Prop Protection
```

Comment:
Use lua style comment --

Reply:
Copied this from the old module without noticing, whoops